### PR TITLE
upx: add page

### PR DESCRIPTION
**Pull request by viztor**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/upx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Compress excutable:
+
+`upx {{file}}`
+
+- Decompress excutable:
+
+`upx -d {{file}}`
+
+- Detailed Help
```
Comment (pxgamer) [inline pages/common/upx]: This description just needs a colon at the end.

File: pages/common/upx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# upx
```
Comment (pxgamer) [inline pages/common/upx]: Looks like this page just needs a `.md` extension.

**Comment by agnivade** [comment]
Thanks @viztor - please add `.md` extension to the file and update the PR. The linter does not check any non .md files. Hence it has not caught any errors.

**Comment by pxgamer** [comment]
I've applied the fixes so that this is reviewable. 👍

File: pages/common/upx.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# upx
+
+> Compress or expand executables.
```
Comment (agnivade) [inline pages/common/upx.md]: Should we change it to `Compress or decompress executables` to maintain consistency with the later description ?

Comment (pxgamer) [reply pages/common/upx.md]: Not sure how I missed that typo. 🤔 But yes, probably would be good. 👍

Comment (pxgamer) [reply pages/common/upx.md]: Made this change in 2122b5b451e03afdecf9ac90aec4d7f6f170738b, should be fine now :+1:

**Comment by agnivade** [comment]
Merging this now. Please feel free to comment here if anybody had any comments on the PR.

**Pull request merged**